modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -1362,6 +1364,49 @@ public void onError(@NotNull Throwable e) {
         });
     }
 
+    /**
+     * Performs {@link RaftGroupService#changePeersAsync(java.util.List, long)} on a provided raft group service.
+     * Retry mechanism is applied to repeat {@link RaftGroupService#changePeersAsync(java.util.List, long)} if previous one
+     * failed with some exception.
+     *
+     * @param raftGroupServiceSupplier Raft groups service.
+     * @return Function which performs {@link RaftGroupService#changePeersAsync(java.util.List, long)}.
+     */
+    public BiFunction<List<Peer>, Long, CompletableFuture<Void>> changePeersAsync(Supplier<RaftGroupService> raftGroupServiceSupplier) {
```

Comment:
private?

Reply:
made package-private, it's needed to make it possible to call it from `org.apache.ignite.internal.table.distributed.TableManagerTest`